tests/state/test_state_api.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from remoulade.api.main import app
+from remoulade.state import StateNamesEnum
+
+
+@pytest.fixture
+def api_client(state_middleware):
+    app.config["TESTING"] = True
+    app.config["backend"] = state_middleware.backend
+    with app.test_client() as client:
+        yield client
+    app.config["TESTING"] = False
```

Comment:
I think we don't really need `app.config["TESTING"] = False`. As the real falsk app will never be started after the end of a test.